skbio/stats/composition.py

```diff
@@ -838,6 +884,12 @@ def ancom(table, grouping,
     if (table.isnull()).any().any():
         raise ValueError('Cannot handle missing values in `table`.')
 
+    for percentile in percentiles:
+        if not 0.0 <= percentile <= 100.0:
+            raise ValueError('Percentiles must be in the range [0, 100], %r '
+                             'was provided.' % percentile)
+
```

Comment:
Now, what if the user gives the following percentiles

```python
percentiles = [0, 50, 10, 100]
```
Should a sort be performed, or should an error be thrown?